src/test/java/com/google/api/codegen/testdata/discoveries/ruby/ruby_bigquery.v2.json.baseline

```diff
@@ -121,38 +98,31 @@ puts JSON.pretty_generate(response)
 #    https://console.developers.google.com/apis/api/bigquery
 # 2. This sample uses Application Default Credentials for authentication.
 #    If not already done, install the gcloud CLI from
-#    https://cloud.google.com/sdk/ and run
+#    https://cloud.google.com/sdk and run
 #    `gcloud beta auth application-default login`
 # 3. Install the Ruby client library and Application Default Credentials
 #    library by running `gem install google-api-client` and
 #    `gem install googleauth`
+
 require 'googleauth'
 require 'google/apis/bigquery_v2'
 require 'json'
 
 service = Google::Apis::BigqueryV2::BigqueryService.new
 
-# Use Application Default Credentials for authentication when running locally.
-# For more information, see:
-# https://developers.google.com/identity/protocols/application-default-credentials
 service.authorization = \
     Google::Auth.get_application_default(['https://www.googleapis.com/auth/cloud-platform'])
 
-
-# TODO: Change placeholder below to desired parameter value for the `list_datasets` method:
-
 # Project ID of the datasets to be listed
-project_id = '{MY-PROJECT-ID}'
-
+project_id = ''  # TODO: Update placeholder value.
 
-# Perform the call
 datasets = service.fetch_all(items: :datasets) do |token|
   service.list_datasets(project_id, page_token: token)
 end
 
-datasets.each do |dataset_list._datasets|
```

Comment:
Was there a special case here for Ruby, like in C#? cf. https://github.com/googleapis/toolkit/commit/b543cf665f5a8c44e56e5e5e42fbc3f5e7755467#diff-010e22cc85bc3c135a473e9305a78ba2R430